code/obj/item/flamethrower.dm

```diff
@@ -2,10 +2,26 @@
 CONTAINS:
 GETLINEEEEEEEEEEEEEEEEEEEEE
 (well not really but it should)
+A Flamethrower
+A Flamethrower that uses a backpack fuel/gas tank
+A Flamethrower backpack fuel/gas tank
+A Flamethrower in various states of assembly
 
 */
-
-/obj/item/flamethrower/
+#define FLAMER_DEFAULT_TEMP 700 KELVIN
+#define FLAMER_BACKTANK_TEMP 1000 KELVIN + T0C
+#define FLAMER_MIN_TEMP T0C
+#define FLAMER_MAX_TEMP 1000 KELVIN + T0C
+#define FLAMER_DEFAULT_CHEM_AMT 10
+#define FLAMER_BACKTANK_CHEM_AMT 25
+#define FLAMER_MIN_CHEM_AMT 5
+#define FLAMER_MAX_CHEM_AMT 20
+#define FLAMER_MODE_LOOSE 1
+#define FLAMER_MODE_TIGHT 2
+#define FLAMER_MODE_SINGLE 3
+#define FLAMER_MODE_BACKTANK 4
```

Comment:
if these are not used elsewhere, undef them at the end of the file

Reply:
Done!